plugins/workbench-impl/src/components/Workbench.svelte

```diff
@@ -84,7 +87,16 @@
   })
 
   let spaceQuery: QueryUpdater<Space> | undefined
+  let notificationQuery: QueryUpdater<SpaceNotifications> | undefined
```

Comment:
You need to have one query for all user spaces to show notifications, right now it is only active space and it is not good, since you need to show badges for all of them.

Reply:
I hope we will hide nav bar